sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamSource.scala

```diff
@@ -88,7 +88,7 @@ class FileStreamSource(
   }
 
   /**
-   * Returns the next batch of data that is available after `start`, if any is available.
```

Comment:
This doc should be updated, following `Source.getBatch()`'s doc change from `Returns the next batch of data that is available after start, if any is available.` to `Returns the data that is between the offsets (start, end].`